proposals/0475-tuple-syntax.rst

```diff
@@ -94,8 +102,9 @@ Proposed Change Specification
      -- ...
      data Tuple64 ... = (...)
 
-#. Export the following definitions from ``GHC.Exts`` and ``GHC.Prelude``. These replace
-   the existing tuple definitions (in ``GHC.Classes``) today. (Note that ``(...) =>`` is special syntax, and does not
+#. Replace the existing constraint tuples in ``GHC.Classes`` with the following
+   definitions and export them from ``Data.Tuple.Experimental`` and ``Prelude.Experimental``. ::
```

Comment:
Same comment.  Don't talk about replacing or about `GHC.Classes`.  Those are implementation matters. Simply write down the definitions visible through `Data.Tuple.Experimental`.